src/lib/statistics/statistics_objects/single_bin_histogram.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "abstract_histogram.hpp"
+#include "types.hpp"
+
+namespace opossum {
+
+/**
+ * Histogram with a single bin.
+ * This histogram type is intended for statistics where only minimal information is known or required (e.g. in Tests).
```

Comment:
What advantage does this have over the generic histogram that justifies having another class?

Reply:
Was great for less typing in tests, I introduced `GenericHistogram::with_single_bin` which provides the same convenience and removed SingleBinHistogram